pennylane/pulse/hardware_hamiltonian.py

```diff
@@ -191,89 +193,66 @@ def circuit_local(params):
 
             amplitude = lambda p, t: p * jnp.sin(jnp.pi * t)
             phase = jnp.pi / 2
-            detuning = 3 * jnp.pi / 4
-            H_d = qml.pulse.drive(amplitude, phase, detuning, wires)
 
-            dev = qml.device("default.qubit.jax", wires=wires)
+            H_d = qml.pulse.drive(amplitude, phase, wires)
+
+            # detuning term
+            H_z = qml.dot([-3*np.pi/2/4]*len(wires), [qml.PauliZ(i) for i in wires])
 
+            dev = qml.device("default.qubit.jax", wires=wires)
             @qml.qnode(dev, interface="jax")
             def circuit(params):
-                qml.evolve(H_i + H_d)(params, t=[0, 10])
+                qml.evolve(H_i + H_z + H_d)(params, t=[0, 10])
                 return qml.expval(qml.PauliZ(1))
 
         >>> params = [2.4]
         >>> circuit(params)
-        Array(0.94301294, dtype=float64)
+        Array(0.6962041, dtype=float64)
         >>> jax.grad(circuit)(params)
-        [Array(0.59484969, dtype=float64)]
-
+        [Array(1.75825695, dtype=float64)]
     """
     if isinstance(wires, int):
         wires = [wires]
```

Comment:
Lillian brought this up in my PR, but this should be updated to accommodate for custom wire labels